### PR TITLE
Fixed borders in Operations/Index template

### LINKED ISSUE #16295: Ugly black lines below the headers of main page
### Describe the bug

Ugly black lines below the headers

### To Reproduce

1. Go to main page

### Screenshots

image

### Server configuration

- phpMyAdmin version: 5.1.0-dev

https://demo.phpmyadmin.net/master/index.php

### PR DESCRIPTION
**Pull request by rajat315315**
### Description

Please describe your pull request.
The problem was due to .card > .list-group border.
It was fixed by changing the class in template to .list-group-noborder & attaching a separate scss for it.

Fixes #16295 

Before submitting pull request, please review the following checklist:

- [x] Make sure you have read our [CONTRIBUTING.md](https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md) document.
- [x] Make sure you are making a pull request against the correct branch. For example, for bug fixes in a released version use the corresponding QA branch and for new features use the _master_ branch. If you have a doubt, you can ask as a comment in the bug report or on the mailing list.
- [x] Every commit has proper `Signed-off-by` line as described in our [DCO](https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO). This ensures that the work you're submitting is your own creation.
- [x] Every commit has a descriptive commit message.
- [x] Every commit is needed on its own, if you have just minor fixes to previous commits, you can squash them.
- [x] Any new functionality is covered by tests.

### CODE REVIEW

**Review by williamdes — APPROVED** [review]
LGTM but maybe someone else has some ideas on this diff

File: templates/home/index.twig [hunk 1]
```diff
@@ -31,7 +31,7 @@
               <div class="card-header">
                 {% trans 'General settings' %}
               </div>
-              <ul class="list-group list-group-flush">
+              <ul class="list-group-noborder list-group-flush">
```
Comment (mauriciofauth) [inline templates/home/index.twig:34]: We don't want to add a new class. You should fix the `list-group` class instead.

**Review by MauricioFauth — CHANGES_REQUESTED** [review]
Looks like it's an issue with Bootstrap 4.5. See https://github.com/phpmyadmin/phpmyadmin/issues/16295#issuecomment-670164871.

**Comment by mauriciofauth** [comment]
Thanks for your contribution. This was a Bootstrap bug. Fixed by upgrading Bootstrap to 4.5.2.

**Pull request closed without merging**